states/huntsman/twilight_flat_fielding.py

```diff
@@ -0,0 +1,71 @@
+"""
+State to handle the taking of calibration frames (evening and morning).
+"""
+
+
+def get_cameras(pocs):
+    '''
+    Get lists of narrow and broad band cameras.
+    '''
+    narrow_band_cameras = list()
+    broad_band_cameras = list()
+    for cam_name, cam in pocs.observatory.cameras.items():
+
+        # This is a hack. There should be a narrow property in the config...
+        if cam.filter_type.lower().startswith('ha'):
+            narrow_band_cameras.append(cam_name)
+        else:
+            broad_band_cameras.append(cam_name)
+
+    return narrow_band_cameras, broad_band_cameras
+
+
+def wait_for_twilight(pocs):
+    '''
+    Wait for twilight. Temporary solution until something better is found.
+    '''
+    pocs.logger.debug('Waiting for twilight...')
+    while pocs.is_safe(horizon='flat'):
+        # Twilight when Sun between flat and focus horizons.
+        twilight = not pocs.is_dark(horizon='focus')
+        if not twilight:
+            pocs.sleep(delay=pocs._safe_delay)
```

Comment:
I don't think the single use, throwaway variable `twilight` is adding anything here, and I think it's actually clearer if you remove it, along with the double negative (two `not`).

```suggestion
        # If it's still too dark to take flats then wait some more.
        if pocs.is_dark(horizon='focus'):
            pocs.sleep(delay=pocs._safe_delay)
```

Reply:
I included it because you said it wasn't clear enough that we were waiting for twilight.